### DISCUSSION TITLE
No access to official website in China?

### DISCUSSION THREAD

**Discussion by Cyberhan123** [react-hook-form/react-hook-form#5261] [Q&A] [answered]
have mirror website？

**Comment by bluebill1049** [comment]
What would need to get this done? That's so sad we are blocked...

**Comment by bluebill1049** [comment]
Could you please try again? We are getting amazing support from Vercel.

**Comment by wangzuo** [comment, marked as answer]
I happened to find this through twitter and I came to host a build on our china server https://reacthookform.caitouyun.com (https://github.com/caitouyun/react-hook-form-mirror) , hope its helpful.

**Reply by bluebill1049** [reply]
Awesome! thank you. The issue got resolved by Vercel's engineer as well. 🙏
